external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaTestUtils.scala

```diff
@@ -507,3 +507,4 @@ class KafkaTestUtils(withBrokerProps: Map[String, Object] = Map.empty) extends L
     }
   }
 }
+
```

Comment:
nit: let's remove this unless you've fixed checkstyle here.

Reply:
Please ignore this if it's invalid. I reviewed against the last commit and not sure why Github says "Outdated".